comps/imagegen/triton/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+ARG TRITON_VERSION=24.04
+
+FROM nvcr.io/nvidia/tritonserver:${TRITON_VERSION}-py3 AS triton
+
+FROM base
+
+ARG MODEL_NAME=stability
+
+RUN wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2204/x86_64/cuda-keyring_1.1-1_all.deb && \
```

Comment:
I need to check this one too.

Reply:
I can take a stab and building the triton containers from source - but I am not sure where that code would live. Probably not in this repo. And who would host it?

The Nvidia-distributed triton server container DOES contain a bunch of extra stuff we don't need.